crates/sui-tls/src/verifier.rs

```diff
@@ -42,32 +39,28 @@ impl Allower for AllowAll {
     }
 }
 
-/// HashSetAllow restricts keys to those that are found in the member set. non-members will not be
-/// allowed.
+/// AllowPublicKeys restricts keys to those that are found in the member set. non-members will
+/// not be allowed.
 #[derive(Debug, Clone, Default)]
-pub struct HashSetAllow {
-    inner: ValidatorAllowlist,
+pub struct AllowPublicKeys {
+    inner: Arc<ArcSwap<BTreeSet<Ed25519PublicKey>>>,
```

Comment:
Is it easy to store `Ed25519PublicKeyAsBytes` instead?

Reply:
I could, but that seems worse, because then every time `Allower::allowed` is called, it would have to use `bincode::serialize` to convert the `&Ed25519PublicKey` to `Ed25519PublicKeyAsBytes`?

What's the motivation here?

Reply:
During TLS verification on client, the server public key bytes is first extracted from the server certificate and then converted to `Ed25519PublicKey` (implemented in `public_key_from_certificate()`). I'm hoping we can avoid this bytes -> `Ed25519PublicKey` conversion all together. So we can just pass `&[u8]` to the allower. Ok to just leave a TODO.